app/screens/emoji_picker/index.tsx

```diff
@@ -46,7 +49,7 @@ const EmojiPickerScreen = ({closeButtonId, componentId, onEmojiPress}: Props) =>
             componentId={componentId}
             contentStyle={style.contentStyle}
             initialSnapIndex={1}
-            footerComponent={PickerFooter}
+            footerComponent={isTablet ? undefined : PickerFooter}
```

Comment:
why this here?

Reply:
We don't want to pass any footerComponent here, because the footer component is already handled inside the body when it is not a tablet. And the `PickerFooter` use a hook that can only be called inside the BottomSheet.